resources/route53-health-checks.go

```diff
@@ -16,24 +16,33 @@ func ListRoute53HealthChecks(sess *session.Session) ([]Resource, error) {
 	params := &route53.ListHealthChecksInput{}
 	resources := make([]Resource, 0)
 
-	resp, err := svc.ListHealthChecks(params)
-	if err != nil {
-		return nil, err
-	}
+	getHealthChecks := func() *bool {
```

Comment:
Please use [`aws.Bool`](https://docs.aws.amazon.com/sdk-for-go/api/aws/#Bool) here.